source/collimation.f90

```diff
@@ -1310,15 +1085,6 @@ subroutine collimate_parseInputLine(inLine, iLine, iErr)
     end if
     call chr_cast(lnSplit(2), dowritetracks, iErr)
 
-  case("CERN")
-    if(nSplit /= 2) then
-      write(lerr,"(a,i0)") "COLL> ERROR CERN expects 1 value, got ",nSplit-1
-      write(lerr,"(a)")    "COLL>       CERN true|false"
-      iErr = .true.
-      return
-    end if
-    call chr_cast(lnSplit(2), cern, iErr)
```

Comment:
don't you want to print an error here, like you do at `BEAM_NUM`?

Reply:
Because the flag was never added to the manual for the new block, so there is no reason te expect anyone ever ued it.

Reply:
ok